keyboards/mechlovin/adelais_en_ciel/adelais_en_ciel.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2020 Team Mechlovin'
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "adelais_en_ciel.h"
+led_config_t g_led_config = { {
```

Comment:
```suggestion

#ifdef RGB_MATRIX_ENABLE
led_config_t g_led_config = { {
```

Reply:
this pcb is rgb pcb, so i think we don't need need to define RGB_MATRIX_ENABLE

Reply:
It is, but somebody may want to ... *shudders* disable the RGB matrix on the board. 

And if they do, the compile will fail if this isn't behind a preprocessor ifdef.